project/Dependencies.scala

```diff
@@ -2,27 +2,29 @@ import sbt._
 
 object Version {
   val logback   = "1.2.3"
-  val mockito   = "1.16.23"
-  val scalaTest = "3.2.3"
+  val mockito   = "3.2.4.0-M1"
+  val scalaTest = "3.2.4-M1"
```

Comment:
ScalaTest 3.2.4 is available in Maven Central
https://repo1.maven.org/maven2/org/scalatest/scalatest_3.0.0-RC1/3.2.4/